src/jit/assertionprop.cpp

```diff
@@ -2852,16 +2852,15 @@ GenTreePtr Compiler::optAssertionProp_LclVar(ASSERT_VALARG_TP assertions, const
     }
 
     BitVecOps::Iter iter(apTraits, assertions);
-    unsigned        index = 0;
-    while (iter.NextElem(&index))
+    for (int i = iter.NextElem(); i != -1; i = iter.NextElem())
     {
```

Comment:
Another pattern would be:
```
int i;
while ((i = iter.NextElem()) != -1)
```
which, IMO, might be preferable, to avoid duplicating `iter.NextElem()`.